src/components/atoms/Notification.tsx

```diff
@@ -0,0 +1,48 @@
+import { Notification, Icon } from "@ledgerhq/react-ui";
+import { CustomContentProps } from "notistack";
+import React from "react";
+
+// NOTE: https://notistack.com/features/customization#custom-variant-(typescript)
+declare module "notistack" {
+  // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+  interface VariantOverrides {
+    errorNotification: {
+      errorType: string;
+    };
+  }
+}
+
+const warningBadge = (
+  <Notification.Badge
+    color="warning.c50"
+    backgroundColor="warning.c100"
+    size={26}
+    icon={<Icon name="Warning" size={20} />}
+  />
+);
+
+type ErrorNotificationProps = CustomContentProps & {
+  errorType: string;
+};
+
+export const ErrorNotification = React.forwardRef<
+  HTMLDivElement,
+  ErrorNotificationProps
+>((props, ref) => {
+  const { id, errorType, message } = props;
+
+  return (
+    <div ref={ref} key={id}>
```

Comment:
From https://notistack.com/features/customization#best-practices
Should we use this ?
```suggestion
    <SnackbarContent ref={ref} {...other}>
```

Reply:
Yup used it and works fine, also removed the forced `height` props passed

Reply:
https://github.com/LedgerHQ/wallet-connect-live-app/assets/5050709/5f3cc278-5cce-479a-bfd4-a193f84940d5